src/pages/search.tsx

```diff
@@ -229,6 +168,9 @@ const Search: NextPage<SearchProps> = ({ translations }): JSX.Element => {
         setCurrentPage(1);
       }
 
+      dispatch({ type: addSearchHistoryRecord.type, payload: debouncedSearchQuery });
+      logTextSearchQuery(debouncedSearchQuery, SearchQuerySource.SearchPage);
+
```

Comment:
Nit-picking: this can also be DRY'd since it's common between the search page and search drawer